lib/index.js

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console, no-process-exit */
+
+const spawn = require('cross-spawn');
+const path = require('path');
+const Spinner = require('cli-spinner').Spinner;
+const fetchAppDefinitionSettings = require('./prompt-app-definition');
+const createAppDefinition = require('./create-app-definition');
+
+const command = process.argv[2];
+const appFolder = process.argv[3];
+
+const mainCommand = 'npx create-contentful-app';
+
+function onSuccess(spinner, folder) {
+  spinner.stop(true);
+
+  console.log();
+  console.log(`Success! Created "${folder}" in ~/${folder}.`);
```

Comment:
Is "in ~/${folder}" part true? I'd just say "Created (full path here)"

Reply:
I must've missed some changes I made last evening when doing the folder migration 🤦